core/src/main/java/com/devonfw/tools/solicitor/componentinfo/scancode/FilteredScancodeComponentInfoProvider.java

```diff
@@ -93,8 +93,8 @@ public void setMinLicensefileNumberOfLines(int minLicensefileNumberOfLines) {
    * Read scancode information for the given package from local file storage.
    *
    * @param packageUrl The package url of the package
-   * @param curationDataSelector identifies which source should be used for the curation data. <code>null</code>
-   *        indicates that the default should be used.
+   * @param curationDataSelector identifies which source should be used for the curation data.If the value of
```

Comment:
The handling of the special value "none" actually is missing in the implementation. It should be done close to lines 156-157.
Alternatively (instead of having the logic here and in ComponentInfoCuratorImpl.curate()) the handling of the value "none" could be moved to CurationProvider.findCurations() (and be implemented in all implementations of CurationProvider)